app/src/main/java/com/example/benchproject/domain/movie/details/entity/MovieDetailsModel.kt

```diff
@@ -10,4 +12,16 @@ data class MovieDetailsModel(
     val rating: Double,
     val releaseDate: String,
     val imageUrl: String
-)
+) {
+    fun mapToUI() = MovieDetails(
```

Comment:
function name shouldn't be renamed already?

Reply:
I changed it in all possible places from `mapToUI` to `toUi`